2020-02-Spring/3_trees/README.md

```diff
@@ -45,21 +45,41 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. Deepest Leaves Sum
 
-Source: TODO :bug:
+Source: https://leetcode.com/problems/deepest-leaves-sum/
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given a binary tree, return the sum of values of its deepest leaves.
+Example 1:
+
+![Images](./Images/Deepest_Leaves_Sum.PNG)
+Constraints:
+
+- The number of nodes in the tree is between 1 and 10^4.
+- The value of nodes is between 1 and 100.
 
 #### Example Input
 
-If the problem is simple enough, remove this section. TODO :bug:
+Input: root = [1,2,3,4,5,null,6,7,null,null,null,null,8]
+Output: 15
 
 #### Function Signature
 
-TODO :bug:
+/**
+ * Definition for a binary tree node.
+ * struct TreeNode {
+ *     int val;
+ *     TreeNode *left;
+ *     TreeNode *right;
+ *     TreeNode() : val(0), left(nullptr), right(nullptr) {}
+ *     TreeNode(int x) : val(x), left(nullptr), right(nullptr) {}
+ *     TreeNode(int x, TreeNode *left, TreeNode *right) : val(x), left(left), right(right) {}
+ * };
+ */
+ 
+ int deepestLeavesSum(TreeNode* root){}
 
```

Comment:
```suggestion

**C++:**
```c++
  Definition for a binary tree node.
  struct TreeNode {
      int val;
      TreeNode *left;
      TreeNode *right;
      TreeNode() : val(0), left(nullptr), right(nullptr) {}
      TreeNode(int x) : val(x), left(nullptr), right(nullptr) {}
      TreeNode(int x, TreeNode *left, TreeNode *right) : val(x), left(left), right(right) {}
  };
 
 int deepestLeavesSum(TreeNode* root){}
```c++

```
Instead of using a multi-line comment, using the triple backtick ``` would turn a block of things into a neat block of things. If it doesn't make sense, refer to the [GitHub markdown page](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks) for more info.